etc/lfcd.fish

```diff
@@ -11,17 +11,8 @@
 #
 # You may put this in a function called fish_user_key_bindings.
 
-function lfcd
-    set tmp (mktemp)
+function lfcd --wraps="lf" --description="lf - Terminal file manager (changing directory on exit)"
     # `command` is needed in case `lfcd` is aliased to `lf`
-    command lf -last-dir-path=$tmp $argv
-    if test -f "$tmp"
-        set dir (cat $tmp)
-        rm -f $tmp
-        if test -d "$dir"
-            if test "$dir" != (pwd)
-                cd $dir
-            end
-        end
-    end
+    command lf -print-last-dir "$argv" \
+        | read -l dir && test -n "$dir" && cd "$dir" || true
```

Comment:
I'm don't know what the advice is for testing whether a directory exists before trying to `cd` inside, but anyway I have found the following to work, which is simpler:

```fish
cd (command lf -print-last-dir $argv)
```

Reply:
My goal here was to preserve the original semantics. *Personally* I agree with you though, because I don't know what kind of errors can make this variable come out empty or why would we want to silence it in such case. Should we change it the way you suggested?

Reply:
I think it's fine since [`lfcd.sh`](https://github.com/gokcehan/lf/blob/master/etc/lfcd.sh) also does this (was changed recently in #1402), and I haven't seen any complaints since.

Reply:
Done 👍